interactive/interactive.go

```diff
@@ -35,6 +40,17 @@ func OpenInBrowser(url string) error {
 	return err
 }
 
+func WSL2() (bool, error) {
+	if runtime.GOOS != "linux" {
+		return false, nil
+	}
+	out, err := exec.Command("cat", "/proc/version").CombinedOutput()
+	if err != nil {
+		return false, errors.Wrapf(err, "detecting WSL2: %s", string(out))
+	}
+	return bytes.Contains(out, []byte("WSL2")), nil
+}
```

Comment:
I would love to move this to https://github.com/efficientgo/core. What do you think? cc @matej-g @bwplotka

Reply:
Also considering adding context support there to enable quick work cancellation. Not sure whether it would be valuable.